tests/test_integration_misc.py

```diff
@@ -284,3 +285,38 @@ def test_empty_case_with_a_pattern_run_generate(self):
         self.assertKpetProduces(
             kpet_run_generate, assets_path,
             stdout_matching=r'.*<job>\s*HOST\s*suite1\s*case1\s*</job>.*')
+
+    def test_preparation_tasks_are_added(self):
+        """Test source-matching with a specific case"""
+        assets = {
+            "index.yaml": """
+                host_type_regex: ^normal
+                host_types:
+                    normal:
+                        tasks: tasks.xml
+                recipesets:
+                    rcs1:
+                      - normal
+                arches:
+                    - arch
+                trees:
+                    tree:
+                        template: tree.xml
+                suites:
+                    - suite1.yaml
+            """,
+            "suite1.yaml": SUITE_BASE.format(1) + """
+                    - name: case1
+                      max_duration_seconds: 600
+                      host_type_regex: normal
```

Comment:
You don't need to specify `host_type_regex` for a case, if it's the same as `host_type_regex` in the top `index.yaml` - that's the default.

Reply:
So, could you remove it?